netbox/dcim/views.py

```diff
@@ -1786,6 +1786,12 @@ class DeviceBulkDeleteView(generic.BulkDeleteView):
     table = tables.DeviceTable
 
 
+class DeviceBulkRenameView(generic.BulkRenameView):
+    queryset = Device.objects.prefetch_related('tenant', 'site', 'rack', 'device_role', 'device_type__manufacturer')
```

Comment:
We can omit `prefetch_related()` here since we're only dealing with the object names.

```suggestion
    queryset = Device.objects.all()
```

Reply:
Done.